cctp_primitives/src/bit_vector/compression.rs

```diff
@@ -203,6 +240,7 @@ mod test {
     fn generate_random_bit_vector(seed: u64) -> Vec<u8> {
         let mut random_generator = rand::rngs::StdRng::seed_from_u64(seed);
         let bit_vector_size: u16 = random_generator.gen();
+        assert!(bit_vector_size as usize <= MAX_DECOMPRESSION_SIZE);
```

Comment:
u16 is for sure less than `MAX_DECOMPRESSION_SIZE`
But we may generate even `u32` value and take the modulus of `MAX_DECOMPRESSION_SIZE`.
In other case, if somebody will change `let bit_vector_size: u16 = random_generator.gen();` it may lead to random assertion during the tests.

Reply:
Fixed.